unit_testing/test_math.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+# pytest unit tests for spinalcordtoolbox.process_seg
+
+# TODO: add test with known angle (i.e. not found with fitting)
+# TODO: test empty slices and slices with two objects
+
+from __future__ import absolute_import
+import sys
+import os
+import pytest
+import math
+import numpy as np
+
+from spinalcordtoolbox.utils import __sct_dir__
+sys.path.append(os.path.join(__sct_dir__, 'scripts'))
+from spinalcordtoolbox import process_seg
+from spinalcordtoolbox.centerline.core import ParamCenterline
+
+from create_test_data import dummy_segmentation
+
+
+# Define global variables
+VERBOSE = 0  # set to 2 to save files
+DEBUG = False  # Set to True to save images
+
+
+dict_test_orientation = [
+    {'input': 0.0, 'expected': 0.0},
+    {'input': math.pi, 'expected': 0.0},
+    {'input': -math.pi, 'expected': 0.0},
+    {'input': math.pi / 2, 'expected': 90.0},
+    {'input': -math.pi / 2, 'expected': 90.0},
+    {'input': 2 * math.pi, 'expected': 0.0},
+    {'input': math.pi / 4, 'expected': 45.0},
+    {'input': -math.pi / 4, 'expected': 45.0},
+    {'input': 3 * math.pi / 4, 'expected': 45.0},
+    {'input': -3 * math.pi / 4, 'expected': 45.0},
+    {'input': math.pi / 8, 'expected': 22.5},
+    {'input': -math.pi / 8, 'expected': 22.5},
+    {'input': 3 * math.pi / 8, 'expected': 67.5},
+    {'input': -3 * math.pi / 8, 'expected': 67.5},
+    ]
+
+
+# noinspection 801,PyShadowingNames
+@pytest.mark.parametrize('test_orient', dict_test_orientation)
+def test_fix_orientation(test_orient):
+    assert process_seg.fix_orientation(test_orient['input']) == pytest.approx(test_orient['expected'], rel=0.0001)
```

Comment:
It confuses me that this function takes in rads and outputs degrees. I would have made it take in rads, output rads, and called it "normalize_orientation".